Components/avatar.html

```diff
@@ -21,139 +21,129 @@
-                    Preview
-                </div>
-               
-                <div class="example-div">
-                    <img class="avatar tiny" src="https://images.unsplash.com/photo-1608889825103-eb5ed706fc64?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=880&q=80" alt="avatar-1">
-
-                    <img class="avatar normal" src="https://images.unsplash.com/flagged/photo-1577351285836-19ff13f8e615?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80" alt="avatar-2">
-
-                    <img class="avatar huge" src="https://images.unsplash.com/photo-1568514328861-5465017e40fc?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=623&q=80" alt="avatar-3">
-                </div>
-
-                <div class="horizontal-line"></div>
-
-             
+            <div class="content-subtitle">
+                Avatar is where a user's profile photo is displayed. They used to be available in square shape however
+                they are now mostly circular. Avatars are used in every website. The main purpose of avatar is to
+                provide identity to the user.
             </div>
-        </main>
-    </div>
- 
+
+            <div class="horizontal-line"></div>
+
+            <div class="content-heading">Profile Avatar</div>
+
+            <div class="content-subtitle">
+                Profile avatar can be created by using the class <code>avatar</code>, inside an <code>img</code> tag
+            </div>
+
+            <iframe class="code-snippet"
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=737.25&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522example-div%2522%253E%250A%2509%250A%2520%2520%253Cimg%2520class%253D%2522avatar%2522%2520src%253D%2522image%2520url%2522%2520alt%253D%2522avatar-1%2522%253E%250A%2520%2520%250A%2520%2520%253Cimg%2520class%253D%2522avatar%2522%2520src%253D%2522image%2520url%2522%2520alt%253D%2522avatar-2%2522%253E%250A%2520%2520%250A%2520%2520%253Cimg%2520class%253D%2522avatar%2522%2520src%253D%2522image%2520url%2522%2520alt%253D%2522avatar-3%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%253C%252Fdiv%253E"
+                style="width: 950px; height: 360px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+
+            <div class="content-heading">Preview</div>
+
+            <div class="example-div">
+                <img class="avatar"
+                    src="https://images.unsplash.com/photo-1608889825103-eb5ed706fc64?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=880&q=80"
```

Comment:
As the image tag is self-closing do use /> for it, it's a good practice otherwise can cause unexpected errors in future

Reply:
I tried that but whenever I pull it from github it adds a self closing tag

Reply:
![image](https://user-images.githubusercontent.com/89568557/154011632-3705a4f8-f883-4668-a4cd-478e908c40ca.png)
It is not showing on vs code but it is seen on github

Reply:
I thought that from html5 onwards there is no need to add /> to a self closing element